src/Forms/FormField.php

```diff
@@ -1223,9 +1224,11 @@ public function Type()
      * @param bool $result
      * @param Validator $validator
      * @return bool
+     * @deprecated 5.4.0 Will be replaced by the `updateValidate` extension hook
      */
     protected function extendValidationResult(bool $result, Validator $validator): bool
     {
+        Deprecation::noticeWithNoReplacment('5.4.0', 'Will be replaced by the `updateValidate` extension hook');
```

Comment:
The method doesn't get replaced with the extension hook - if I'm calling this method in my custom form field I don't now need to implement an extension class, I need to call `extend()` directly.
```suggestion
        Deprecation::noticeWithNoReplacment('5.4.0', 'Use extend("updateValidate") instead');
```

Reply:
Please also update the phpdoc and the changelog

Reply:
`extend("updateValidate")` does not exist in CMS 5, so we cannot give this recommendation.

Reply:
`use extend() directly instead` then

Reply:
I don't understand how you'd use extend() directly in CMS 5, or at least why you would

The thing being replaced here has a bool + a Validator for params.  The functionality that's replacing it in CMS 6 has a ValidationResult param instead.

Presumably you'd need to implement something sort of convoluted in CMS 5 for this to work ... only to then replace it straight away in CMS 6?

Reply:
> I don't understand how you'd use extend() directly in CMS 5, or at least why you would

`extendValidationResult()` is ultimately an abstraction around `$this->extend('updateValidationResult', $result, $validator);`

You can just call that directly.

In CMS 5 you'll get a deprecation warning if you're calling `extendValidationResult()`. One way to silence that is to call `extend()` directly instead of calling `extendValidationResult()`.

In CMS 6 that will also be the solution - calling `extend()`, since `extendValidationResult()` will no longer exist.

Reply:
Updated